livy/src/main/resources/interpreter-setting.json

```diff
@@ -118,6 +118,11 @@
         "defaultValue": "1000",
         "description": "Max number of Spark SQL result to display."
       },
+      "zeppelin.livy.spark.sql.truncate": {
+        "propertyName": "zeppelin.livy.spark.sql.truncate",
+        "defaultValue": "true",
+        "description": "If true, truncate strings greater than 20 characters."
```

Comment:
How about `zeppelin.livy.spark.sql.field.truncate` ?  This is for truncating each field in table.